how-to/customize-workspace/client/src/framework/platform/interopbroker.ts

```diff
@@ -395,6 +412,42 @@ export function interopOverride(
 				throw new Error(RESOLVER_TIMEOUT);
 			}
 		}
+
+		public async fdc3HandleOpen(
+			fdc3OpenOptions: { app: PlatformApp | string; context: OpenFin.Context },
+			clientIdentity: OpenFin.ClientIdentity
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		): Promise<any> {
+			if (fdc3OpenOptions?.app === undefined || fdc3OpenOptions?.app === null) {
+				logger.error("A request to fdc3.open did not pass an fdc3OpenOptions object");
+				throw new Error(OPEN_APP_NOT_FOUND);
+			}
+
+			const requestedId =
+				typeof fdc3OpenOptions.app === "string"
+					? fdc3OpenOptions.app
+					: fdc3OpenOptions.app.appId || fdc3OpenOptions.app.name;
+			const openAppIntent: OpenFin.Intent = {
+				context: fdc3OpenOptions.context,
+				name: "OpenApp",
+				metadata: {
+					target: requestedId
+				}
+			};
+			logger.info(
+				`A request to Open has been sent to the platform by uuid: ${clientIdentity?.uuid}, name: ${clientIdentity?.name}, endpointId: ${clientIdentity.endpointId} with passed context:`,
+				fdc3OpenOptions.context
+			);
+			try {
+				const result = await this.handleFiredIntent(openAppIntent);
+				return { appId: result.source };
+			} catch (intentError) {
+				if (intentError?.message !== undefined && intentError.message === NO_APPS_FOUND) {
```

Comment:
Could be simplified to `intentError?.message === NO_APPS_FOUND`

Reply:
will update